src/main/webapp/app/course/dashboards/assessment-dashboard/assessment-dashboard.component.html

```diff
@@ -164,24 +164,15 @@ <h2 *ngIf="isExamMode">
 
                     <td>
                         <ng-container *ngIf="!exercise.teamMode; else teamsLink">
-                            <ng-container *ngIf="!isTestRun; else testRun">
+                            <ng-container>
                                 <a
-                                    [routerLink]="['/course-management', courseId, 'exercises', exercise.id, 'tutor-dashboard']"
+                                    [routerLink]="getTutorDashboardLinkForExercise(exercise)"
```

Comment:
I guess this method should be renamed to `getTutorDashboardLinkForExercise(exercise)`